### DISCUSSION TITLE
[Feature] Enable casting for Android app.

### DISCUSSION THREAD

**Discussion by heliotropeRD** [immich-app/immich#9829] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm reopening this closed request. One of the primary use cases for Immich is to keep my photos on my machine, but be allowed to share them with people I want to see them. As many people have casting capable devices at their homes, being able to cast to their systems is independent of the existing Android TV app. 

Request: Please add casting support to the android app. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thank you for your request. Duplicate of #7030
